### PR TITLE
Feature/add password criteria 470

### PR DESCRIPTION
**Pull request by cschu**

### CODE REVIEW

File: datameta/security/__init__.py [hunk 1]
```diff
@@ -83,9 +83,35 @@ def check_expiration(expiration_datetime: Optional[datetime]):
...
 
-def verify_password(s):
-    if len(s) < 10:
-        return "The password has to have a length of at least 10 characters."
+def verify_password(password):
+    PW_MINIMUM_LENGTH = 10
+    PW_MINIMUM_UPPERCASE = 1
+    PW_MINIMUM_LOWERCASE = 1
+    PW_MINIMUM_DIGITS = 1
+    PW_MINIMUM_PUNCTUATION = 1
```
Comment (lkuchenb) [inline datameta/security/__init__.py:91]: As discussed, please move to app settings

**Review by lkuchenb — CHANGES_REQUESTED** [review]
Thanks, just some minor remarks, see inline comments

**Comment by cschu** [comment]
@lkuchenb back to you

**Review by lkuchenb — APPROVED** [review]
@cschu First time we're using non-str values in the appsettings, triggered a bug (the admin page for application settings / the GET endpoint for appsettings didn't load anymore). Fixed in my added commits. Good to merge from my end.

**Comment by cschu** [comment]
> @cschu First time we're using non-str values in the appsettings, triggered a bug (the admin page for application settings / the GET endpoint for appsettings didn't load anymore). Fixed in my added commits. Good to merge from my end.

I remember there was an issue couldn't recall it was an interface issue. Good that it's solved now..

**Pull request merged**